library/src/main/java/com/novoda/downloadmanager/lib/Query.java

```diff
@@ -31,14 +31,14 @@
      */
     public static final int ORDER_DESCENDING = 2;
 
-    private static final String ORDER_BY_LIVENESS = String.format("CASE batch_status "
+    private static final String ORDER_BY_LIVENESS = String.format("CASE status "
```

Comment:
Why the change from `batch_status` to `status` ? 
Should this be then  [`Downloads.Impl.COLUMN_STATUS`](https://github.com/novoda/download-manager/blob/master/library/src/main/java/com/novoda/downloadmanager/lib/Downloads.java#L204) ?